RetroBar/Taskbar.xaml

```diff
@@ -13,6 +13,9 @@
         Deactivated="Taskbar_Deactivated"
         AllowDrop="True"
         Style="{DynamicResource TaskbarWindow}">
+    <WindowChrome.WindowChrome>
+        <WindowChrome x:Name="Chrome" CaptionHeight="0" CornerRadius="0"/>
+    </WindowChrome.WindowChrome>
```

Comment:
I don't think this is needed as it stands, and it currently breaks ClearType, so let's remove it.